### ISSUE TITLE
nevegrad 'instrumentation' problem

### ISSUE THREAD

**Issue by gvignoud** [brian-team/brian2modelfitting#35]
Hi, when installing using pip, it installs nevergrad 0.4.1post3 which is not compatible.
It raises, 
from nevergrad import instrumentation as inst
ImportError: cannot import name 'instrumentation'

Going back to nevegrad 0.3.0 solved the problem for me.
Best,

**Comment by mstimberg** [comment]
Yes, this is a known issue and already "fixed" in the master branch (by requiring nevergrad 0.3). We will do a release soon so that it is fixed when installing from pip. I'll leave the bug report open to make this clearer to other users who run into the same problem.

**Comment by mstimberg** [comment]
Fixed via #68
